src/components/TaskListView/TaskListView.css

```diff
@@ -0,0 +1,110 @@
+.done {
+    text-decoration: line-through;
+}
+
+.check-task {
+    height: 25px;
+    width: 25px;
+    background-color: transparent;
+    background-image: url('empty-check.png');
+    border: none;
+}
+
+.checked {
+    background-color: transparent;
+    background-image: url('check-mark.png');
+    border: none;
+}
+
+.bin {
+    height: 25px;
+    width: 25px;
+}
+
+.trash {
+    border: none;
+    background-color: transparent;
+    display: flex;
+    justify-content: center;
+}
+
+.trash:hover, .check-task:hover {
+    cursor: pointer;
+    transform: scale(1.1);
+}
+
+.center-clm {
+    display: flex;
+    justify-content: center;
+    padding: 7px
+}
+
+.complete-header {
+    text-align: center;
+}
+
+table {
+    width: clamp(400px, 60%, 800px);
+    /* property: clamp(minimum, preferred, maximum); */
+    margin-top: 20px;
+    border-collapse: collapse;
+    border-radius: 10px;
+    overflow: hidden;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.15);
+}
+
+thead, tbody tr {
+    display: table;
+    width: 100%;
+    table-layout: fixed;
+}
+
+tbody {
+    display: block;
+    max-height: 511px;
+    overflow-y: auto;
+}
+
+th, td {
+    padding: 15px;
+    text-align: left;
+}
+
+th {
+    background-color: rgba(66, 43, 212, 0.3);
+}
+
+tr:nth-child(odd) {
+    background-color: rgba(217, 213, 246, 0.5);
+}
+
+tr:nth-child(even) {
+    background-color: rgba(251, 251, 254, 0.5);
+}
+
+.no-tasks {
+    padding: 10px;
+    text-align: center;
+    font-size: 1.5em;
+}
+
+::-webkit-scrollbar {
+    width: 8px; 
+}
+
+::-webkit-scrollbar-thumb {
+    background-color: #B3AAEE; 
+    border-radius: 4px; 
+}
+
+::-webkit-scrollbar-track {
+    background-color: #FBFBFE; 
+}
+
+::-webkit-scrollbar-corner {
+    background-color: #D9D5F6; 
+}
+
+::-webkit-scrollbar-thumb:hover {
+    background-color: #422BD4; 
+}
```

Comment:
I notice that `TableRow.js` does not have a corresponding css file. For organizational clarity, can we please have a `TableRow.css` that has all the styling for classes contained in TableRow.js?